share/jupyter/kernels/xpython/kernel.json.in

```diff
@@ -6,4 +6,5 @@
       "{connection_file}"
   ],
   "language": "python"
+  "metadata": { "debugger": true }
```

Comment:
Looks like there is a missing coma one the line above?

Reply:
oops ;) Thanks for catching it!

Reply:
JSON linter in the :eyes: :)